tests/integration/targets/aci_aep/tasks/main.yml

```diff
@@ -51,6 +69,22 @@
     - nm_add_aep.current[0].infraAttEntityP.attributes.dn == 'uni/infra/attentp-ansible_test'
     - nm_add_aep.current[0].infraAttEntityP.attributes.name == 'ansible_test'
     - cm_add_aep.proposed.infraAttEntityP.attributes.name == nm_add_aep.proposed.infraAttEntityP.attributes.name == cm_add_aep.sent.infraAttEntityP.attributes.name == nm_add_aep.sent.infraAttEntityP.attributes.name == 'ansible_test'
+    - nm_add_aep2.current[0].infraAttEntityP.children[0].infraProvAcc.attributes.name == 'provacc'
+    - nm_add_aep3.current[0].infraAttEntityP | length == 1
+
+- name: Change AEP infra(normal mode)
+  cisco.aci.aci_aep:
+    <<: *aep_present
+    aep: ansible_test2
+    infra_vlan: false
+  register: nm_add_aep_true_to_false
+
+- name: Change AEP infra 2(normal mode)
+  cisco.aci.aci_aep:
+    <<: *aep_present
+    aep: ansible_test2
+    infra_vlan: false
+  register: nm_add_aep_true_to_false2
 
 - name: Add AEP again (check_mode)
   cisco.aci.aci_aep: *aep_present
```

Comment:
Between lines 97-98, please add a test case to add again aep: ansible_test3 as you are not testing what happens when you re-add an aep with infra_vlan = false.
Make sure to register the variable and test for is not changed.